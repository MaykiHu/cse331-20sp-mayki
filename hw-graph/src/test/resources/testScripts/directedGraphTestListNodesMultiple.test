#Create a graph
CreateGraph graph1

#Create a pair of nodes
CreateNode 1
CreateNode 2

#Add a pair of nodes to the graph
AddNode graph1 n1
AddNode graph1 n2

#List nodes in graph
ListNodes graph1