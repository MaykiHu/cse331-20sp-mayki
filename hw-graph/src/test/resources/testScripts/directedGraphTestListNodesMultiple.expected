#Create a graph
created graph graph1

#Create a pair of nodes
created node n1
created node n2

#Add a pair of nodes to the graph
added node n1 to graph1
added node n2 to graph1

#List nodes in graph
graph1 contains: n1 n2